### PR TITLE
style(data-management): Make action card look good

### PR DESCRIPTION
**Pull request by Twixes**
## Problem

Noticed those definition popovers we have look a bit wonky, particularly if any text gets a bit longer… or if "All events" is used.

## Changes

| Before | After |
| --- | --- |
| before | after |

Also removed some ants.

## How did you test this code?

Added the action definition popover to Storybook.

### CODE REVIEW

File: frontend/__snapshots__/filters-taxonomicpopover--taxonomic-string-popover-one-category.png
Comment (mariusandra) [inline frontend/__snapshots__/filters-taxonomicpopover--taxonomic-string-popover-one-category.png:1]: This looks odd:

image

File: frontend/src/lib/components/DefinitionPopover/DefinitionPopover.scss [hunk 1]
```diff
@@ -87,8 +89,7 @@
     }
 
     .definition-popover-timemeta {
-        display: inline;
-        margin-top: 4px;
+        margin-top: 8px;
         margin-bottom: 4px;
```
Comment (mariusandra) [inline frontend/src/lib/components/DefinitionPopover/DefinitionPopover.scss:93]: Should these margins here and above be `rem`-s instead?

**Review by mariusandra — APPROVED** [review]
Looks good and makes sense. Some nits/comments inline.

File: frontend/__snapshots__/filters-taxonomicpopover--taxonomic-string-popover-one-category.png
Comment (Twixes) [reply frontend/__snapshots__/filters-taxonomicpopover--taxonomic-string-popover-one-category.png:1]: I think that's a GitHub bug, looks good on my ~~machine~~ browser 🤔

File: frontend/src/lib/components/DefinitionPopover/DefinitionPopover.scss (hunk 1 above)
Comment (Twixes) [reply frontend/src/lib/components/DefinitionPopover/DefinitionPopover.scss]: Hm, I was about to leave them as-is, but I suppose consistency is good – so changed them all to rems

**Pull request merged**